timbot.py

```diff
@@ -149,4 +162,30 @@ def run_timbot():
 
 
 if __name__ == '__main__':
-    main()
+
+    # load configuration data
+    try:
+        with open('config.yaml', 'r') as file:
+            config = yaml.load(file)
+    except Exception as e:
+        print("Error loading configuration data: ", e)
+        sys.exit()
+
+    # connect to database
+    try:
+        conn = mysql.connector.connect(user=config['mysql']['user'],
+                                       password=config['mysql']['password'],
+                                       host=config['mysql']['host'],
+                                       database=config['mysql']['database'])
+                                     
+    except mysql.connector.Error as err:
+        if err.errno == errorcode.ER_ACCESS_DENIED_ERROR:
+            print("Something is wrong with your user name or password")
+        elif err.errno == errorcode.ER_BAD_DB_ERROR:
+            print("Database does not exist")
+        else:
+            print(err)
+        sys.exit()
+    
+    else:
```

Comment:
Why is the `else` clause needed? You're `sys.exit()` if an error has been encountered.

Reply:
I understand how this could be confusing to a more junior dev such as yourself, but I will change it to assuage your concerns

Reply:
no fancy words please